xml/yast2_sw.xml

```diff
@@ -336,21 +336,20 @@
      </mediaobject>
     </informalfigure>
     <para>
-     To perform a search, just write a term in the <guimenu>Package
-     Name</guimenu> input field and press the <guimenu>Search</guimenu> button.
+     To perform a search, just enter a term in the <guimenu>Package
+      Name</guimenu> input field and press <keycap function="enter"/> or the
+     <guimenu>Search</guimenu> button.
      &yast; then contacts the &scc; and, after a few seconds, shows the results
-     in the table, including the module or extension each package belongs to.
+     in a table, including the module or extension each package belongs to.
+     Select a package to see additional details.
     </para>
-    <!-- FIXME cwickert 2020-06-26: Usability for selecting packages is really
-     bad and subject to criticism, see https://bugzilla.suse.com/1165913
-     Behavior will probably change, make sure the following is still correct.
-    -->
     <para>
-     Select one or more packages for installation by clicking on the
-     corresponding table row and pressing the <guimenu>Toggle
-     Selection</guimenu> button. Alternatively, you can just double-click on a
-     row. If the package belongs to a not activated module or extension, &yast;
-     asks for confirmation to activate it. Click <guimenu>Next</guimenu>,
+     To install a package, select it and press the <guimenu>Toggle
+      Selection</guimenu> button. Alternatively, you can just double-click on
+     the package. If it belongs to an unregistered module or extension, &yast;
+     asks for confirmation to register it. Packages are marked for installation
+     with <guimenu>&plus;</guimenu> in the <guimenu>Status</guimenu> column.
+     Selected all packages you want to install, click <guimenu>Next</guimenu>,
```

Comment:
```suggestion
     Select all packages you want to install, click <guimenu>Next</guimenu>,
```

Reply:
Thanks for catching this. I'll fix it when we have the "few seconds" sorted out.